### DISCUSSION TITLE
manual for the work with tables in github -

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#55380] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hi i need to have a good manual for the creating of tables in github

found this:  https://github.github.com/gfm/#tables-extension-

```

| f\|oo  |
| ------ |
| b `\|` az |
| b **\|** im |



```
and this 

```
Example 199
| abc | defghi |
:-: | -----------:
bar | baz

```

**Comment by airtower-luna** [comment]
What you found there is the specification, which should describe exactly what works, but might not be super easy to read. :wink: 

Here's the matching manual: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

**Reply by Seunfola** [reply]
You can as well use markdown language to create GitHub tables

**Reply by airtower-luna** [reply]
That *is* markdown. :wink:
